src/SFA.DAS.EmployerUsers.Web/Global.asax.cs

```diff
@@ -28,6 +28,8 @@ protected void Application_Start()
             AreaRegistration.RegisterAllAreas();
             GlobalConfiguration.Configure(WebApiConfig.Register);
             FilterConfig.RegisterGlobalFilters(GlobalFilters.Filters);
+            GlobalFilters.Filters.Add(new System.Web.Mvc.AuthorizeAttribute { Roles = "das-support-portal" });

```

Comment:
If the controllers have their own attributes to control this role access, is this needed?